dprint.json

```
@@ -7,7 +7,8 @@
         "**/*.{yml,yaml}"
     ],
     "excludes": [
-        "pnpm-lock.yaml"
+        "pnpm-lock.yaml",
+        "./changeset/*"
```

Pretty sure it should be
```suggestion
        ".changeset/*"
```

---

ooops